### PR TITLE
Add more text content and set only the tab that is clicked in the projects page to the color of the corresponding content.

### PR DESCRIPTION
**Pull request by nikhitashanker**
Currently, no content is shown on the projects page before a tab is clicked. In this change, by default the content for the first tab is shown. Additionally, only the tab button that is clicked most recently has its color matching the corresponding content. More text was added to the HTML elements. When a dropdown button is clicked on the main page, we toggle between a plus sign and a minus sign on the right end of the button to make it more intuitive that the button needs to be clicked to see the content.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -76,4 +84,20 @@
...
+
+#recommender-system {
+  background-color: peachpuff;
+}
+
+#swe-dev-portal {
+  background-color: oldlace;
+}
+
+#five-minutes-a-day {
```
Comment (liangqigong) [inline portfolio/src/main/webapp/style.css:97]: Not in alphabetic order?

Comment (nikhitashanker) [reply portfolio/src/main/webapp/style.css:97]: Within each of the blocks, I believe the declarations are in order. Do the blocks themselves also need to be in order?

Comment (hlz03) [reply portfolio/src/main/webapp/style.css:97]: No, the blocks doesn't have to be in order.

Comment (nikhitashanker) [reply portfolio/src/main/webapp/style.css:97]: Ack.

**Pull request merged**